contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/index.vue

```diff
@@ -240,15 +240,13 @@
       undo: 'Undo',
       cancel: 'Cancel',
       close: 'Close',
-      duplicateSelectedButton: 'Duplicate selected items on clipboard',
-      moveSelectedButton: 'Move selected items',
-      deleteSelectedButton: 'Remove selected items from clipboard',
-      removingItems: 'Removing {count, plural,\n =1 {# item}\n other {# items}}...',
-      removedFromClipboard: 'Removed from clipboard',
-      creatingClipboardCopies:
-        'Creating {count, plural,\n =1 {# copy}\n other {# copies}} on clipboard...',
-      copiedItemsToClipboard:
-        'Copied {count, plural,\n =1 {# item}\n other {# items}} to clipboard',
+      duplicateSelectedButton: 'Make a copy',
+      moveSelectedButton: 'Move',
+      deleteSelectedButton: 'Delete',
+      removingItems: 'Deleting from clipboard...',
+      removedFromClipboard: 'Deleted from clipboard',
+      creatingClipboardCopies: 'Copying in clipboard...',
```

Comment:
We missed to correct this and use `to` instead of `in`, right? To avoid conundrums `on vs. in`... 
Please change to `Copying to clipboard...`, to also be consistent to other strings below.

Reply:
Nvm, my bad!